dev/package.json

```diff
@@ -1,8 +1,11 @@
 {
-  "name": "Flapjack",
+  "name": "syrup-test",
   "version": "1.0.0",
   "description": "A simple test application for syrup.",
   "main": "gulpfile.js",
+  "homepage": "http://allenai.github.io/syrup/",
+  "repository": "allenai/syrup",
+  "license": "GPL-2.0",
```

Comment:
Hmm, great catch.  I'm not super knowledgeable on os licensing and `npm` complains without it.

Would MIT be more permissive?  What license would you recommend?